tests/boards/mec15xxevb_assy6853/i2c_mainline/src/main.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/i2c.h>
+#include <zephyr.h>
+#include <ztest.h>
+
+#define I2C_DEV_NAME DT_LABEL(DT_ALIAS(i2c1))
+
+/* command to configure port direction of NXP PCA95xx */
```

Comment:
It's not a command. It's a register.

Reply:
this is a command, it means that when sending this command byte, it will write data into register 6 and 7 of nxp pca95xx device